sr_robot_launch/config/controller_spawner.yaml

```diff
@@ -1,8 +1,3 @@
-controller_configs:
-  rh:
-    rh_trajectory_controller: rh_trajectory_controller.yaml
-  lh:
-    lh_trajectory_controller: lh_trajectory_controller.yaml
 controller_groups:
```

Comment:
why got rid of this hand controllers?

Reply:
because I dont want those params to be automatically dumped to parameter server, I am doing that manually within the launch files with a proper values depending on the setup.

Reply:
cool